kubernetes/samples/charts/apache/default-sample.md

```diff
@@ -0,0 +1,38 @@
+# Apache Load Balancer default sample
+## Configure Apache Webtier as Load Balancer for WLS Domains
+In this section we will demonstrate how to use Apache webtier to handle traffic to backend WLS domains.
+
+### 1. Install WebLogic Server Domain
+Now we need to prepare some backends for Apache to do load balancing.
+
+Create a WebLogic Server domain:
+- Under default namespace.
+- Domain name is 'domain1'.
+- Deploy a webapp  with url context 'testwebapp'.
```

Comment:
Do we have the webapp jar in the samples?
Do we have the command to deploy the app?

Reply:
What I used is the integration test script to create domain and deploy web application. It call WLST command to deploy app.  The testwebapp.war and python script can be found in /shared directory in the managed server pod.

Reply:
For each step in the README, we need to have the supporting files accessible, either as part of the sample itself, or pointers to the location where the files can be accessed.  We also need to document the command or points to another doc where the command is documented.